jobs/build-podman-os.Jenkinsfile

```diff
@@ -122,6 +130,24 @@ def cosa_memory_request_mb = 4.5 * 1024 as Integer
 def ncpus = ((cosa_memory_request_mb - 512) / 1536) as Integer
 
 
+node {
```

Comment:
there is already a `node {` declaration higher up in this file.

Reply:
just me being a groovy noob :| . btw, there's another one below as well. Should I look at reusing either one or do you have a preference?

Reply:
hmm weird. yeah, just re-use the one below

Reply:
Hmm, thinking about this further, I should grab podman version and upstream commit from the podman-next copr rpm that's fetched in the build process https://github.com/containers/podman-machine-os/blob/main/podman-image-daily/Containerfile#L31